Setara/Setara/App/ViewController/HomeViewController.swift

```diff
@@ -74,7 +74,7 @@ class HomeViewController: UIViewController {
   @objc func nameFromTextField(){
     tableView.reloadData()
     guard let name = textFieldName.text else { return }
-    let addNewParticipant = CoreDataManager.shared.createParticipant(name: name)
+    _ = CoreDataManager.shared.createParticipant(name: name)
```

Comment:
kenapa di hilangin name object nya?

Reply:
Kemarin ada alert the object never used trus gua click fix pan